PocketKit/Sources/PocketKit/Settings/SettingsView.swift

```diff
@@ -69,12 +69,14 @@ struct SettingsForm: View {
     private struct settingsCredits: View {
         let appVersion = Bundle.main.infoDictionary!["CFBundleVersion"] as! String
         let buildNumber = Bundle.main.infoDictionary!["CFBundleShortVersionString"] as! String
+        let gitSha = Bundle.main.infoDictionary!["GitSha"] as? String ?? ""
```

Comment:
If we keep nabbing things from the info dictionary, we might want to create a cleaner API for this. But, perhaps something for later.

Reply:
i agree.